src/main/java/com/redhat/management/approval/Logger.java

```diff
@@ -61,6 +61,10 @@ public static Logger getLogger(String httpRequestId) {
     return LOGGER;
   }
 
+  public static Logger getLogger(Object httpRequestId) {
+    return httpRequestId == null ? getLogger() : getLogger(httpRequestId.toString());
```

Comment:
`httpRequestId` can be `null` when passed to `getLogger`, no need special treatment.

Reply:
👍